service/src/main/resources/changelog/changesets/20230118_assemble_refbased_prepopulation.yaml

```diff
@@ -0,0 +1,1597 @@
+databaseChangeLog:
+  - changeSet:
+      id: "method prepopulation for assemble_refbased"
+      author: kpierre
+      changes:
+        - insert:
+            tableName: method
+            columns:
+              - column:
+                  name: method_id
+                  value: 00000000-0000-0000-0000-000000000005
+              - column:
+                  name: name
+                  value: assemble_refbased
+              - column:
+                  name: description
+                  value: assemble_refbased / Target Workflow 2
```

Comment:
wait, sorry, I know I approved this already but I'm actually kind of confused now about what this is -- are "assemble_refbased" and "Target Workflow 2" the same thing?

Reply:
I see that they are - I think I was just thrown off by the PR names. This seems ok to me.

Reply:
If it's confusing then I can get rid of it. Maybe that's better for public preview anyway.